GA_core/ga_classes.py

```diff
@@ -5,26 +5,51 @@ def update(self, generation, best_score):
 
 
 class Genetic_Algorithm:
+    """
+    gene = policy (per SKU)
+
+    chromosome = individual solution (e.g., for 100 SKUs, max 100 policies)
+
+    population = a number of chromosomes
+    """
 
     def __init__(self) -> None:
         self.observers = []
         
         def create_population(self) -> None:
+            """
+            create some number of Individual_Solution objects

```

Comment:
correct. we will add a parameter to indicate the number of **unique solutions** to start the loops